ci/run_ci.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# install all dependencies only once
+set -e
+. helper-functions.sh
+SCRIPT_LOC=$(pwd)
+
+print_header "Installing dependencies"
+
```

Comment:
Lets check passed args and print errors if something wasn't passed like we do in the manager.sh